python/taichi/lang/matrix.py

```diff
@@ -565,6 +565,26 @@ def rotation2d(alpha):
         return ti.Matrix([[ti.cos(alpha), -ti.sin(alpha)],
                           [ti.sin(alpha), ti.cos(alpha)]])
 
+    @staticmethod
+    def var(n, m, dt, shape):
+        import taichi as ti
+        return ti.Matrix(n=n, m=m, dt=dt, shape=shape)
+
+    @staticmethod
+    def cols(columns):
+        import taichi as ti
+        return ti.Matrix(cols=columns)
```

Comment:
Thank for refactor, could you move the codes in if-else-if in `__init__` to here?